iiasa/message_ix
# Extend feature vintage_and_active_years()

This PR extends the features of the function `vintage_and_active_years()` based on issue #571 

## How to review

- Read the diff and note that the CI checks all pass.
- Build the documentation and look at a description of `vintage_and_active_years()`
- Ensure that changes/additions are self-documenting, i.e. that another
  developer (someone like the reviewer) will be able to understand what the code
  does in the future.

## PR checklist
